tracebot_mockup_gazebo/models/Pump/model.sdf

```diff
@@ -0,0 +1,108 @@
+<?xml version='1.0'?>
+<sdf version='1.7'>
+  <model name='Pump'>
+    <link name='link_3'>
+      <inertial>
+        <mass>1</mass>
+        <inertia>
+          <ixx>0.166667</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.166667</iyy>
+          <iyz>0</iyz>
+          <izz>0.166667</izz>
+        </inertia>
+        <pose>0 0 0 0 -0 0</pose>
+      </inertial>
+      <pose>-0 0 0 1.5708 -0 0</pose>
+      <gravity>1</gravity>
+      <self_collide>0</self_collide>
+      <kinematic>0</kinematic>
+      <enable_wind>0</enable_wind>
+      <visual name='visual'>
+        <pose>0 0 0 0 -0 0</pose>
+        <geometry>
+          <mesh>
+            <uri>/home/benrjg/astech_wss/tracebot_mockup_ws/src/tracebot_mockup/tracebot_mockup_description/meshes/pump.stl</uri>
```

Comment:
This and some other `.sdf` files still have hardcoded paths to files in your machine. I'm not familiar with the SDF format, but I guess there must be a way to avoid hardcoded paths.

Reply:
The hard coded paths and other .world or .sdf related issues have been fixed in the next pull. I only picked up on these when working with the camera and vision in gazebo. The world and model files will then be used, I'll push these shortly

Reply:
I can remove these for now if you'd prefer (or you can remove them in pulling), I'm just not too sure if this will cause deletion when I try to merge the next branch

Reply:
Nevermind, if they're going to be used eventually let's not bother with fixing it here.